Frontend/Pages/TaskList/HtbDocument/Preview.cshtml.cs

```diff
@@ -164,6 +165,7 @@ public async Task<IActionResult> OnGet(string id)
                     HintText =
                         "If you add comments, they'll be included in the latest Ofsted judgement section of your project template.",
                     Urn = Project.Urn,
+                    AddOrEditAdditionalInformation = false,
```

Comment:
Think you flagged this in an earlier commit but had conflicts so it's crept it back in! I'll remove and repush..